supporting-blog-content/customer-success-example/README.md

```diff
@@ -0,0 +1,35 @@
+Example of AI Conversational Search using Elasticsearch, OpenAI and Vercel. 
+
+# Load the data
+
+Export the following environment variables: 
+- ELASTICSEARCH_URL: You can find it in your Elasticsearch deployment
+- ES_API_KEY: Your Elasticsearch API Key
+
+Run the script ingest.py: `python ingest.py` to index the data. 
+
+
+# Chat application
+
```

Comment:
[Optional] It might be useful to add a small architecture diagram showing the communication between the components, especially since you are pulling the results to Elasticsearch and then passing to OpenAI within your application.